lib/ActivityStreams.js

```diff
@@ -66,6 +66,11 @@ ActivityStreams.prototype = {
           this.send(am.actions.Response("RECENT_BOOKMARKS_RESPONSE", links), worker);
         });
         break;
+      case am.type("RECENT_LIKS_REQUEST"):
```

Comment:
s/LIKS/LINKS/

We should almost revisit the "should every `switch` have a `default` case to catch unexpected things.
I have no idea how/if something like this would be catchable via any linter.
Although we could go the old Adobe Flex route of having constants for event names and stuff. So instead of static strings like "RECENT_LINKS_REQUEST" we'd export a const like `am.responses.RECENT_LINKS_RESPONSE = "RECENT_LINKS_RESPONSE";` which could be whatever string value we want. That way any linter would catch typos.